libsigner/src/session.rs

```diff
@@ -31,7 +31,7 @@ pub trait SignerSession {
     /// connect to the replica
     fn connect(
         &mut self,
-        host: SocketAddr,
+        host: String,
```

Comment:
Nit: Generally you shouldn't take owned wrapper types as input to function calls. It's better to take `&str` here.

```suggestion
        host: &str,
```

Reply:
Good call, I changed instances of `String` arg types to `&str` -- how's it look now?

Reply:
Better, but you missed a few spots. Pointed them out.